.github/workflows/ci.yml

```
@@ -0,0 +1,146 @@
+        key: ${{ runner.os }}-pnpm-store-${{ hashFiles('**/pnpm-lock.yaml') }}
+        restore-keys: |
+          ${{ runner.os }}-pnpm-store-
+    
+    - name: Install dependencies
+      run: pnpm install --frozen-lockfile
+    
+    - name: Build application
+      run: pnpm build
+      env:
+        VITE_APP_ENV: production
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)